config/initializers/prometheus.rb

```diff
@@ -1,45 +1,60 @@
 # The below is heavily cribbed from ministryofjustice/laa-apply-for-criminal-legal-aid
 return unless ENV.fetch("ENABLE_PROMETHEUS_EXPORTER", "false") == "true"
 
+# require 'prometheus_exporter/client'
 require "prometheus_exporter/server"
+require "prometheus_exporter/instrumentation"
+require "prometheus_exporter/middleware"
 
 DEFAULT_PREFIX = "ruby_".freeze
+DEFAULT_BIND_ADDRESS = "0.0.0.0".freeze
+DEFAULT_PORT = 9394
 
 # We are running puma in single process mode, so this is safe
 # If we move to multi process mode, we will have to run the
 # exporter process separately (`bundle exec prometheus_exporter`)
 def start_prometheus_server
   server = PrometheusExporter::Server::WebServer.new(
-    bind: "0.0.0.0", port: ENV.fetch("PROMETHEUS_EXPORTER_PORT", 9394).to_i,
+    bind: ENV.fetch("PROMETHEUS_EXPORTER_HOST", DEFAULT_BIND_ADDRESS),
+    port: ENV.fetch("PROMETHEUS_EXPORTER_PORT", DEFAULT_PORT).to_i,
     verbose: ENV.fetch("PROMETHEUS_EXPORTER_VERBOSE", "false") == "true"
   )
 
   server.start
-
-  true
+  server
 rescue Errno::EADDRINUSE
   warn "[PrometheusExporter] Server port already in use."
   false
 end
 
-return unless start_prometheus_server
+Rails.logger.info("[PrometheusExporter] Starting server....")
+server = start_prometheus_server
+return unless server
 
-require "prometheus_exporter/instrumentation"
-require "prometheus_exporter/middleware"
+Rails.logger.info("[PrometheusExporter] server started on #{JSON.parse(server.to_json).fetch_values('port', 'bind')}!")
+
+# TODO: do we need this - check
+# PrometheusExporter::Client.default = PrometheusExporter::LocalClient.new(collector: server.collector)
```

Comment:
Shall we just remove these lines?

Reply:
yeah, need to try this out as tidy up on submit too, as this does not look like it is needed, nor the `require 'sidekiq/api'`